orchestrator/ethereum_gravity/src/logic_call.rs

```diff
@@ -310,6 +301,12 @@ impl LogicCallSkips {
     }
 }
 
+impl Default for LogicCallSkips {
+    fn default() -> Self {
+        Self::new()
```

Comment:
or can just remove `new` and add it here (it can just be constructed using `::default()` instead of `::new()`)

Reply:
```
warning: function cannot return without recursing
   --> ethereum_gravity/src/logic_call.rs:305:5
    |
305 |     fn default() -> Self {
    |     ^^^^^^^^^^^^^^^^^^^^ cannot return without recursing
306 |         Self::default()
    |         --------------- recursive call site
    |
```

seems its triggering some recursion error

Reply:
What I meant was to move what's inside `pub fn new()` to `fn default()`, and remove the `new()` fn definition (and whenever it was used, replace it with `default()`)